package.json

```diff
@@ -3,12 +3,17 @@
   "version": "0.0.1",
   "type": "module",
   "scripts": {
-    "test": "node spec/equality-with-mountebank.spec.js"
+    "test": "node spec/all.spec.js"
   },
   "bin": {
     "run-amock": "bin/run"
   },
-  "keywords": ["http", "mock", "govuk", "GOV.UK"],
+  "keywords": [
+    "http",
+    "mock",
+    "govuk",
+    "GOV.UK"
+  ],
   "author": "Natalie Carey <natalie.carey@digital.cabinet-office.gov.uk>",
```

Comment:
I know you've written most of the code in this repo but I think this should be `GOV.UK Pay`, similar to other NPM libs we maintain (which are also majority one person).

Reply:
Thanks, I've removed it.  I wasn't sure what our standard was.